images/Makefile

```diff
@@ -38,7 +40,7 @@ cockroachdb.%:
 minio.%:
 	@$(MAKE) -C minio PLATFORM=$*
 
-do.build.images.%: ceph.% ceph-toolbox.% cockroachdb.% minio.% ;
+do.build.images.%: $(foreach i,$(IMAGES), $(i).%);
```

Comment:
jenkins seems to be hitting an error during cleanup
```
=== cleaning images for build-88c6720f
Makefile:43: *** target pattern contains no '%'.  Stop.
image.mk:100: recipe for target 'clean.build' failed
make[1]: *** [clean.build] Error 2
Makefile:151: recipe for target 'clean' failed
make: *** [clean] Error 2
```

Reply:
Hmm. I'll look into it.